lib/dialogs/about/index.tsx

```diff
@@ -1,125 +1,151 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Modal from 'react-modal';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import SimplenoteLogo from '../../icons/simplenote';
 import CrossIcon from '../../icons/cross';
 import TopRightArrowIcon from '../../icons/arrow-top-right';
 import Dialog from '../../dialog';
+import { closeDialog } from '../../state/ui/actions';
+
+import * as S from '../../state';
 
 const appVersion = config.version; // eslint-disable-line no-undef
 
-export class AboutDialog extends Component {
+type DispatchProps = {
+  closeDialog: () => any;
+};
+
+type Props = DispatchProps;
+
+export class AboutDialog extends Component<Props> {
   static propTypes = {
-    actions: PropTypes.object.isRequired,
-    dialog: PropTypes.object.isRequired,
-    requestClose: PropTypes.func.isRequired,
+    themeClass: PropTypes.string.isRequired,
   };
 
   render() {
-    const { dialog, requestClose } = this.props;
+    const { closeDialog, themeClass } = this.props;
     const thisYear = new Date().getFullYear();
 
     return (
-      <div className="about">
-        <Dialog hideTitleBar onDone={requestClose} title={dialog.title}>
```

Comment:
oh who'da'thunk it? I didn't think about moving the `Modal` back into the dialogs. did this end up feeling better to you than wrapping them in the `<Modal>`?

Reply:
It's a bit repetitive, BUT means the parent doesn't have to load the close function or determine the modal titles, so on the whole, yes. But I'm open to differing opinions.

Reply:
OK I lied I think I like this better https://github.com/Automattic/simplenote-electron/pull/1981/commits/b98f9606cd71bf987fb4f3b627910e80e136f921